unit/test/Set-VSTeamPermissionInheritance.Tests.ps1

```diff
@@ -68,11 +69,8 @@ Describe 'VSTeamPermissionInheritance' {
       Mock _getInstance { return 'https://dev.azure.com/test' }
       Mock _getApiVersion { return '1.0-unitTests' } -ParameterFilter { $Service -eq 'Build' -or $Service -eq 'Release' -or $Service -eq 'Git' }
 
-      # Mock the call to Get-Projects by the dynamic parameter for ProjectName
-      Mock Invoke-RestMethod { return @() } -ParameterFilter { $Uri -like "*_apis/projects*" }
```

Comment:
Yes because of the ProjectCache is moved you can just add this code at the top of a test.

```powershell
# Prime the project cache with an empty list. This will make sure
# any project name used will pass validation and Get-VSTeamProject 
# will not need to be called.
[vsteam_lib.ProjectCache]::Update([string[]]@())
```